lib/cfg/cfg-parsers/llvm-ir.ts

```diff
@@ -83,21 +83,21 @@ export class LlvmIrCfgParser extends BaseCFGParser {
         while (i < fn.end) {
             const match = code[i].text.match(this.labelRe);
             if (match) {
+                const label = match[1];
                 if (bbStart === i) {
-                    // for -emit-llvm the first basic block doesn't have a label, for the ir viewer it does though
                     assert(result.length === 0);
-                    bbStart = i + 1;
+                    // Make the first block name fnName + label
+                    currentName += '\n' + label;
```

Comment:
This is probably ok for now but it occurs to me that making the block name itself fnName + label might not be fully adequate. What if there's a branch to the entry block's label?

Maybe it could make sense to have the entry block appear as its own basic block and just have the function label's block through to it, or maybe we could add special casing for basic blocks with multi-line names during the lookup process.

Reply:
Right, this case didn't occur to me (partly because I never came across it). This PR as-is doesn't make things any worse for this case, but I'd be happy to try and tackle this. Please hold merging it for now.

Reply:
I think I'd prefer *not* to add an artificial function-name-only entry block. Did I understand correctly your first suggestion?
I'll probably be able to make the branch-target search account for multi-line block names.

Reply:
Updated with something along these lines. All node IDs are now strictly labels, and there's a 'prefix' hook used just for display of the entry block.